src/zope/app/apidoc/codemodule/browser/introspector.py

```diff
@@ -95,6 +95,7 @@ class TraversalRoot(object):
 
 
 class Introspector(BrowserView):
+    "Introspector browser view"
```

Comment:
pylint does not have a warning for that. Neither does flake8 as far as I can see:
```
$ flake8 --isolated src/zope/app/apidoc/codemodule/browser/introspector.py
src/zope/app/apidoc/codemodule/browser/introspector.py:33:1: E302 expected 2 blank lines, found 1
src/zope/app/apidoc/codemodule/browser/introspector.py:34:14: E721 do not compare types, use 'isinstance()'
src/zope/app/apidoc/codemodule/browser/introspector.py:111:49: E225 missing whitespace around operator
```

Still, PEP 257 [does say](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring):

> For consistency, always use `"""triple double quotes"""` around docstrings.

I'm horribly inconsistent in my use of `"`, `'` and apparently `"""`, just doing whatever is easiest at the time. I'll try to be more consistent with one-line docstrings. Thanks for pointing it out.

PS: Here's pylint's default output for that same file. I have a number of those warnings disabled by default when I'm working in the zope code base:
```
$ pylint -r no src/zope/app/apidoc/codemodule/browser/introspector.py
************* Module zope.app.apidoc.codemodule.browser.introspector
C:158, 0: Wrong hanging indentation (remove 13 spaces).
                                 name, klass._Class__all_ifaces)
                    |            ^ (bad-continuation)
C:184, 0: Wrong hanging indentation (remove 1 space).
                     val.__doc__ or '',
                    |^ (bad-continuation)
C:185, 0: Wrong hanging indentation (remove 1 space).
                     getParent(self.klassView.context).getPath()),
                    |^ (bad-continuation)
C:187, 0: Wrong hanging indentation (remove 1 space).
                     name, klass._Class__all_ifaces)}
                    |^ (bad-continuation)
C: 33, 0: Invalid function name "getTypeLink" (invalid-name)
C: 33, 0: Missing function docstring (missing-docstring)
R: 38, 4: Consider using ternary (path.replace('.', '/') if importable else None) (consider-using-ternary)
C: 41, 0: Invalid class name "annotationsNamespace" (invalid-name)
W: 44,27: Unused argument 'request' (unused-argument)
C: 47, 4: Missing method docstring (missing-docstring)
W: 47,29: Unused argument 'ignore' (unused-argument)
R: 41, 0: Too few public methods (1/2) (too-few-public-methods)
C: 59, 0: Invalid class name "sequenceItemsNamespace" (invalid-name)
W: 62,27: Unused argument 'request' (unused-argument)
C: 65, 4: Missing method docstring (missing-docstring)
W: 65,29: Unused argument 'ignore' (unused-argument)
R: 59, 0: Too few public methods (1/2) (too-few-public-methods)
C: 72, 0: Invalid class name "mappingItemsNamespace" (invalid-name)
W: 81,27: Unused argument 'request' (unused-argument)
C: 84, 4: Missing method docstring (missing-docstring)
W: 84,29: Unused argument 'ignore' (unused-argument)
R: 72, 0: Too few public methods (1/2) (too-few-public-methods)
C: 92, 0: Missing class docstring (missing-docstring)
R: 92, 0: Too few public methods (0/2) (too-few-public-methods)
C:109, 8: Invalid attribute name "klassView" (invalid-name)
C:115, 4: Missing method docstring (missing-docstring)
C:118, 4: Invalid method name "getBaseURL" (invalid-name)
C:118, 4: Missing method docstring (missing-docstring)
C:121, 4: Invalid method name "getDirectlyProvidedInterfaces" (invalid-name)
C:121, 4: Missing method docstring (missing-docstring)
C:127, 4: Invalid method name "getProvidedInterfaces" (invalid-name)
C:127, 4: Missing method docstring (missing-docstring)
C:130, 4: Invalid method name "getBases" (invalid-name)
C:130, 4: Missing method docstring (missing-docstring)
C:133, 4: Invalid method name "getAttributes" (invalid-name)
C:133, 4: Missing method docstring (missing-docstring)
W:158,39: Access to a protected member _Class__all_ifaces of a client class (protected-access)
C:164, 4: Invalid method name "getMethods" (invalid-name)
C:164, 4: Missing method docstring (missing-docstring)
W:187,27: Access to a protected member _Class__all_ifaces of a client class (protected-access)
C:194, 4: Invalid method name "isSequence" (invalid-name)
C:194, 4: Missing method docstring (missing-docstring)
C:198, 4: Invalid method name "getSequenceItems" (invalid-name)
C:198, 4: Missing method docstring (missing-docstring)
C:212, 4: Invalid method name "isMapping" (invalid-name)
C:212, 4: Missing method docstring (missing-docstring)
C:216, 4: Invalid method name "getMappingItems" (invalid-name)
C:216, 4: Missing method docstring (missing-docstring)
C:230, 4: Invalid method name "isAnnotatable" (invalid-name)
C:230, 4: Missing method docstring (missing-docstring)
C:233, 4: Invalid method name "getAnnotationsInfo" (invalid-name)
C:233, 4: Missing method docstring (missing-docstring)
E:239,26: Too many positional arguments for method call (too-many-function-args)
C: 20, 0: standard import "import inspect" should be placed before "import six" (wrong-import-order)

-----------------------------------
Your code has been rated at 4.77/10
```